workbase/wrappers-bazel/BUILD

```diff
@@ -0,0 +1,22 @@
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
```

Comment:
can we name the package to just be `//workbase/builder` ?

Reply:
oh wait, how many files and folders start with the string `build`? it sounds like there are many now?

Reply:
@haikalpribadi we can discuss it on a call. I didn't want to refactor it together with build integration to separate task scopes